config/index.ts

```diff
@@ -0,0 +1,10 @@
+const BASE_URLS = {
+  prod: 'https://api.id.gov.sg',
+  stag: 'https://api-stg.id.gov.sg',
+  dev: process.env.OVERRIDE_DEV || '',
+}
+const PORT = process.env.PORT || 10000
+const RANDOMNONCE = 'randomnonce'
```

Comment:
Suggestion: shall we use the random nonce returned from `authorizationUrl(...)` instead?

Reply:
Remove hardcoded nonce and uses dynamic nonce in [73aca9f](https://github.com/opengovsg/sgid-test-app/pull/94/commits/73aca9fd057c093722ee6f95ba31f347607f4197).